telemetry/definitions/commonDefinitions.json

```diff
@@ -1627,6 +1627,9 @@
                 {
                     "type": "reason",
                     "required": false
+                },
+                {
+                    "type": "successCount"
```

Comment:
How about successInvocationCount?

Reply:
Is this a required field?

Reply:
sounds good, make more sense with successInvocationCOunt

Reply:
yep, it's required, IMO

Reply:
oh n,o there is no existing type `successInvocationCount`

Reply:
https://github.com/aws/aws-toolkit-common/blob/master/telemetry/definitions/commonDefinitions.json#L374

Reply:
I think `succssfulCount` should be the one we are looking for as per the description except we wanna create another type @leigaol

Reply:
since we might want to use existing type as possible as we can

Reply:
@JadenSimon @justinmk3

Reply:
> use existing type as possible as we can

yes, thank you :)

Reply:
Yeah try to re-use things if possible, just don't force it. If something is different then let it be different.

We haven't really used `successCount` much but I think the original intent was to describe 'compressed' events e.g. instead of emitting 10 events for `s3_uploadObject`, emit 1 event with `successCount` set to 10. If `successCount` doesn't seem too ambiguous for your use-case then go ahead and use it. Otherwise I'd create a new field that is more specific.

Reply:
@JadenSimon @justinmk3, noted, just think of a question, in this PR we added a `required` field `succssCount`, however the plugin versions in the marketplace doesn't have this field yet, will telemetry still consume these data without the required field ?

Reply:
You can make it required. 

If we were talking about the API for a service then making it required would be a breaking change. However, the telemetry backend is agnostic to these definitions and so backwards compatibility is only a concern for people looking at the data and not the service.